internal/uvm/create_wcow.go

```diff
@@ -66,8 +66,7 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 		owner:               opts.Owner,
 		operatingSystem:     "windows",
 		scsiControllerCount: 1,
-		vsmbDirShares:       make(map[string]*vsmbShare),
-		vsmbFileShares:      make(map[string]*vsmbShare),
+		vsmbShares:          make(map[string]*VsmbShare),
```

Comment:
In Go you typically write acronyms/initialisms in either all uppercase or all lowercase. Kevin gets me on this all the time :P 

So instead of VsmbShare, you'd do VSMBShare. 

ref: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I'll do the same for ScsiMount then